pkg/config/config_template.yaml

```diff
@@ -949,6 +949,11 @@ api_key:
   #
   # force_use_http: true
 
+  ## @param transport_type - string - optional - default: auto
+  ## @env DD_LOGS_CONFIG_TRANSPORT_TYPE - string - optional - default: auto
+  ## The transport type to use for sending logs. Possible values are "auto" or "http1".
+  # transport_type: auto
```

Comment:
I'm wondering if the config key should be something `http_protocol` or `http_version` as I thin is more indicative of what we are modifying base on this config. I think `transport` is Go specific things within the http package.